packages/connect-popup/e2e/tests/methods.test.ts

```diff
@@ -76,14 +80,21 @@ filteredFixtures.forEach(f => {
 
         const screenshotsPath = await ensureDirectoryExists(`./e2e/screenshots/${f.url}`);
 
-        const { explorerPage, exploreUrl, browserContext } = await getContexts(
+        const { explorerPage, explorerUrl, browserContext } = await getContexts(
             page,
             url,
             isWebExtension,
         );
         context = browserContext;
 
-        await explorerPage.goto(`${exploreUrl}#/method/${f.url}`);
+        if (connectSrc) {
+            await setConnectSettings(explorerPage, explorerUrl, {
+                trustedHost: false,
+                connectSrc,
+            });
+        }
```

Comment:
ok so we basically stopped using query string and use this connectSrc option instead right?

Reply:
Yes, we need that for the webextension.